ui/src/app/views/workspace/component.css

```diff
@@ -5,6 +5,7 @@
 
 .description-text {
   color: #333;
+  width: 60rem;
```

Comment:
This might be another place (use your own judgement) where a font-size-based size is appropriate. `ex` means "width of a lower-case x" (I think), so you could use something like `80ex`.